web/tools/webpack.config.babel.js

```diff
@@ -15,14 +15,15 @@ const MomentLocalesPlugin = require('moment-locales-webpack-plugin')
 const babelConfig = require('../babel.config.js')
 const fs = require('fs')
 const loadBuildConfig = require('build-configs').default
+const { WEB } = require('build-configs')
 
 const currentYear = new Date().getFullYear()
 
 const SHORT_COMMIT_SHA_LENGTH = 8
 
 // A first performance budget, which should be improved in the future: Maximum bundle size in Bytes; 2^20 = 1 MiB
 // eslint-disable-next-line no-magic-numbers
-const MAX_BUNDLE_SIZE = 1.56 * Math.pow(2, 20)
+const MAX_BUNDLE_SIZE = 1.62 * Math.pow(2, 20)
```

Comment:
Not sure whether we want this, see other PR with same changes.

Reply:
Yes, not sure why this shows up here... Will revert depending on other PR